src/sdk-reference/bulk/import/import.js.md

```diff
@@ -0,0 +1,50 @@
+# Import
+
+```javascript
+import(bulkData, options = {})  ⇒ object
+```
+
+The bulk import allows to save a list of documents into a specific collection (belonging to a specific index). If a subset of the documents fails to save, a PartialError is triggered. The `data` argument passed to the method must specify the set of documents to import and must satisfy the [Elasticsearch Bulk API](https://www.elastic.co/guide/en/elasticsearch/reference/5.4/docs-bulk.html).
```

Comment:
"a `PartialError` error is triggered"

And the workd "PartialError" should be a link leading to this page: https://docs.kuzzle.io/api-documentation/errors#partialerror